### ISSUE TITLE
I found an issue probably many people have same situation like me

### ISSUE THREAD

**Issue by tori-hotaru** [Akegarasu/lora-scripts#3]
Device: win10 / 3060ti
In my training step, i use the notepad++ to change the parameters, however, when i used shell to activate, it gave me this error.(picture 1)
![Q5P10SX7W5TW}G4HD8}W (V](https://user-images.githubusercontent.com/58972377/217157344-2fff2473-7b84-42ff-bfef-954e8894d9c5.png). 
I quickly found that there is something wrong in this train.ps1. Althought the parameter "--learning-rate" has number in it , i dont know why the programme ignore it. So I change the programme like this 
![E(0OA`QZV(6XS@$HZ5 `DC](https://user-images.githubusercontent.com/58972377/217158149-57b03244-27d9-400d-ab6e-9c45a3071267.png)
Fortunately, it worked, but then it reported me another error like this.
![)@J1F(1MWWKV B N$I{`MKQ](https://user-images.githubusercontent.com/58972377/217158568-379c78f8-ba95-4856-9c84-d221d64f0ea2.png)

So is there any better way to solve those problems? I was quite confused.

**Comment by Akegarasu** [comment]
You need to enclose the lr in double quotes. for example,  `$lr = "1e-4"`

**Comment by tori-hotaru** [comment]
No i think it is not the reason, even i only change the path in this programme,  like this.

![$6G8$7M(JH(8) EC(8_L~AL](https://user-images.githubusercontent.com/58972377/217170957-dc8f5c4e-1bcf-4fa9-b9a1-5b061e1be774.png)
![}@Y$IWH@EEC4K0C5DO8N GN](https://user-images.githubusercontent.com/58972377/217170960-80eea334-a41c-4b47-89d9-492ae8bcd5a3.png)
it still had this problem, and it is not only the learning rate paramater invalided, network dim,alpha, all these paramaters cant be recognized by this programme

**Comment by tori-hotaru** [comment]
you can see in the shell error,all the parameter with red line can not be recognized by programme, even number in it
![XSXSZX4LU}K15XU%4MYEH`9](https://user-images.githubusercontent.com/58972377/217173196-3cf25f5b-c782-4fca-9594-58957496b16a.png)

**Comment by Akegarasu** [comment]
This is because your new control character is `lf` and the correct one should be `crlf`. Please do not download the compressed package from github, but use the command in readme to clone this project with submodules.

**Comment by Akegarasu** [comment]
fixed.
